static/bidder-info/adot.yaml

```diff
@@ -1,5 +1,6 @@
 maintainer:
   email: "admin@we-are-adot.com"
+gvlVendorID: 272
```

Comment:
This GVL ID is registered to "A.Mob". Please confirm this is either the same company or if there exists a business relationship.

Reply:
Hello @SyntaxNode,

Yes this is the same company, A.Mob (Adotmob) recently became Adot.

Best regards,